grunt/esbuild/build.js

```diff
@@ -52,6 +52,8 @@ const nodeConfig = {
    * doesn't work at all.
    */
   external: ['ws', 'got'],
+  format: 'cjs',
```

Comment:
If we bundle node.js as an UMD, we will end up with calling `require('got')` in UMD header, which inevitably fails with `[ERR_REQUIRE_ESM]` error since got@12 is pure ESM and can't be `require`d.
So one option is to stop using UMD for node.js bundle and just bundle it as CJS. Then we will only `await import('got')` whenever we actually need it